frontend/src/Navbar.tsx

```diff
@@ -88,25 +88,22 @@ const Navbar: FunctionComponent = () => {
                 onClick={() => setProfileDiv(!profileDiv)}
               />
               {profileDiv && (
-                <div className="absolute right-0 mt-2 w-48 bg-white rounded-md overflow-hidden shadow-xl z-20 mr-2">
-                  <span className="block px-4 py-2 text-sm text-gray-800 border-b hover:bg-gray-200">
+                <div className="absolute right-0 mt-2 w-48 bg-white rounded-md overflow-hidden shadow-xl z-20 mr-2 cursor-pointer">
+                  <a
```

Comment:
i wonder if we can use the `reloadDocument` property in Link to simulate <a href> 

https://reactrouter.com/docs/en/v6/api#link